include/openthread/thread_ftd.h

```diff
@@ -556,13 +556,26 @@ otError otThreadGetNextCacheEntry(otInstance *aInstance, otCacheEntryInfo *aEntr
  * Get the Thread PSKc
  *
  * @param[in]   aInstance   A pointer to an OpenThread instance.
- *
- * @returns A pointer to Thread PSKc
+ * @param[out]  aPskc       A pointer to an `otPskc` to return the retrieved Thread PSKc.
  *
  * @sa otThreadSetPskc
  *
  */
-const otPskc *otThreadGetPskc(otInstance *aInstance);
+void otThreadGetPskc(otInstance *aInstance, otPskc *aPskc);
```

Comment:
Similarly to other keys getters, I think it would be better to leave an option to return an error if retrieving key values is forbidden by security policy on given system.